introduction_to_amazon_algorithms/linear_learner_abalone/Linear_Learner_Regression_csv_format.ipynb

```diff
@@ -14,7 +14,9 @@
     "1. [Introduction](#Introduction)\n",
     "2. [Setup](#Setup)\n",
     "   1. [Exploring the dataset](#Exploring-the-dataset)\n",
-    "3. [Training the Linear Learner model](#Training-the-Linear-Learner-model)\n",
+    "3. [Training the Linear Learner Model](#Training-the-linear-learner-model)\n",
+    "   1. [Training with SageMaker Training](#Training-with-sagemaker-training)\n",
+    "   2. [Training with Automatic Model Tuning (HPO)](#Training-with-automatic-model-tuning-HPO)\n",
```

Comment:
Formatting between first and second subpoints doesn't seem right, at least in ReviewNB.

Reply:
I just double-checked and they look good on the notebook itself. Maybe ReviewNB has some formatting issues when it comes to nested numbered list.